src/theme/components/subnav.ts

```diff
@@ -0,0 +1,169 @@
+import { createMultiStyleConfigHelpers } from "@chakra-ui/styled-system";
+import { StyleFunctionProps } from "@chakra-ui/system";
+
+const { defineMultiStyleConfig, definePartsStyle } =
+  createMultiStyleConfigHelpers([
+    "base",
+    "selectedItem",
+    "borderLine",
+    "outLine",
+    "secondaryActions",
+    "primaryActions",
+  ]);
+
+interface SubNavStyleProps extends StyleFunctionProps {
+  backgroundColor: string;
+  highlightColor: string;
+  isOutlined: boolean;
+}
+const commonStyles = () => ({
+  alignItems: "center",
+  display: "inline-flex",
+  fontWeight: "regular",
+  gap: "xs",
+  height: { base: "44px", md: "unset" },
+  lineHeight: "1.5 !important",
+  px: "s",
+  py: "xxs",
+  marginY: "4px",
+  textDecoration: "none !important",
+  transition: "background-color 0.2s, color 0.2s !important",
+});
+
+const SubNav = defineMultiStyleConfig({
+  baseStyle: definePartsStyle(
+    ({ backgroundColor, highlightColor, isOutlined }: SubNavStyleProps) => {
+      const defaultLabelColor = "ui.typography.body";
+      const highlightOrLinkColor = highlightColor
+        ? `${highlightColor} !important`
```

Comment:
I tried to avoid using !important, but in this case, it's necessary due to multiple components(Buttons, Links, Icons, primary actions and secondary actions) where the colors need to be applied.

Reply:
This was true before but now the component is simpler. Can you try removing more `!important` flags to make sure they are not needed?